code/modules/fish/fishtank.dm

```diff
@@ -194,15 +195,14 @@
 
 	//Check breeding conditions
 	if(fish_count >=2 && egg_count < max_fish)		//Need at least 2 fish to breed, but won't breed if there are as many eggs as max_fish
-		if(food_level > 2 && filth_level <=5)		//Breeding is going to use extra food, and the filth_level shouldn't be too high
+		if(food_level >= 0.2 && filth_level <=5)	//Breeding is going to use extra food, and the filth_level shouldn't be too high
 			if(prob(((fish_count - 2) * 5)+10))		//Chances increase with each additional fish, 10% base + 5% per additional fish
-				egg_count++						//A new set of eggs were laid, increase egg_count
-				egg_list.Add(select_egg_type())		//Add the new egg to the egg_list for storage
-				food_level -= 2						//Remove extra food for the breeding process
+				breed_fish()
+				food_level -= 0.2					//Remove extra food for the breeding process
```

Comment:
No it can't, just as above, since this has a minimum food level of 0.2 required to even reach this line.